app/code/local/LoyaltyLion/CouponImport/controllers/Adminhtml/QuickSetupController.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+require( Mage::getModuleDir('', 'LoyaltyLion_Core') . DS . 'lib' . DS . 'loyaltylion-client' . DS . 'lib' . DS . 'connection.php' );
+
+class LoyaltyLion_CouponImport_Adminhtml_QuickSetupController extends Mage_Adminhtml_Controller_Action
+{
+    private $ll_url = 'https://loyaltylion.com';
```

Comment:
I think this should be camelCase? Not that we have a PHP style guide, but everything else is?